examples/discovery-search-app/cypress/fixtures/components/search_input/autocompletionQuery.json

```diff
@@ -0,0 +1,3397 @@
+      ],
+      "document_passages": [
+        {
+          "passage_text": "The\nserver\ncannot\n<em>have</em>\nsquash\nenabled.\nAll of the\nnodes in\nthe\ncluster\nmust\n<em>have</em>\naccess to\nmount\nthe NFS\nserver\nand <em>have</em>\nread/write\naccess to\nthe\nserver.\n62\n-\n-\n-\n-\nYou must <em>have</em> a cluster-admin account to install Cloud Pak for Data.Ensure\nthat you are logged in.",
+          "start_offset": 2284,
+          "end_offset": 2547,
+          "field": "text"
+        },
+        {
+          "passage_text": "The\nserver\ncannot\n<em>have</em>\nsquash\nenabled.\nAll of the\nnodes in\nthe\ncluster\nmust\n<em>have</em>\naccess to\nmount\nthe NFS\nserver\nand <em>have</em>\nread/write\naccess to\nthe\nserver.\n61\n-\n-\nSecurity requirements\nA Docker registry must be accessible from all of the nodes in the cluster.",
+          "start_offset": 1594,
+          "end_offset": 1851,
+          "field": "text"
+        }
+      ]
+    },
+    {
+      "document_id": "12cefddd-6e95-40d7-8a33-f6efa9d95577",
+      "result_metadata": {
+        "collection_id": "fbafd5e1-9cf1-c24c-0000-016eeb0a5c9f",
+        "document_retrieval_source": "search",
+        "confidence": 0.05165
+      },
+      "enriched_text": [
+        {
+          "entities": [
```

Comment:
seems like a fairly large fixture - are we utilizing any of this in the test?

Reply:
Not in the tests so far. I was planning to keep the whole response handy until all the tests were finished, then cutting out the unused data.